blog/2022-10-26-test-report/index.md

```diff
@@ -76,15 +76,15 @@ With reference to the SLI (Service Level Indicator)/SLO (Service Level Objective
 
 | Status  | SLI                                                                                                                      | SLO                                                                           |
 |---------|--------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------|
-| Offical | P99 latency of Mutating API calls (including POST, PUT, DELETE, PATCH) to a single resource object in the last 5 minutes | P99 <= 1s                                                                     |
-| Offical | P99 latency of non-streaming read-only API calls (including GET and LIST) in the last 5 minutes                          | (a)Scope=resource, P99 <= 1s, (b)Scope=namespace or Scope=cluster, P99 <= 30s |
+| Offical | P99 latency of Mutating API calls (including POST, PUT, DELETE, PATCH) to a single resource object in the last 5 minutes | `P99 <= 1s    `                                                                 |
```

Comment:
Can you remind me why we need to do this update?

Reply:
because of mdx 3 (`P99 <= 1s`) this was an error, I don’t know why, it wrote that I didn’t like “=" and wrapped it in quotes

Reply:
I tried (tested) a solution with escaping these characters, it helped

Reply:
If there are not strong comments, then you can create an issue, I will correct it.

Reply:
> because of mdx 3 (P99 <= 1s) this was an error,  I don’t know why,

What's the original error look like?
I can't figure out if this is a required change for docusaurus v3.

Reply:
I just made a commit with the error I wrote about
https://app.netlify.com/sites/karmada/deploys/656f1aa6ec01f50008d94171

Reply:
I can't see the error :(
Just see following info:
```
8:50:00 PM: [ERROR] Deploy logs are currently unavailable. We are working on resolving the issue.
```

Reply:
**netlify/karmada/deploy-preview** 



2:43:43 PM: [ERROR] Client bundle compiled with errors therefore further build is impossible.
2:43:43 PM: Error: MDX compilation failed for file "/opt/build/repo/blog/2022-10-26-test-report/index.md"
2:43:43 PM: Cause: Unexpected character `=` (U+003D) before name, expected a character that can start a name, such as a letter, `$`, or `_`
2:43:43 PM: Details:
2:43:43 PM: {
2:43:43 PM:   "column": 210,
2:43:43 PM:   "message": "Unexpected character `=` (U+003D) before name, expected a character that can start a name, such as a letter, `$`, or `_`",
2:43:43 PM:   "line": 80,
2:43:43 PM:   "name": "80:210",
2:43:43 PM:   "place": {
2:43:43 PM:     "line": 80,
2:43:43 PM:     "column": 210,
2:43:43 PM:     "offset": 8862,
2:43:43 PM:     "_index": 0,
2:43:43 PM:     "_bufferIndex": 74
2:43:43 PM:   },
2:43:43 PM:   "reason": "Unexpected character `=` (U+003D) before name, expected a character that can start a name, such as a letter, `$`, or `_`",
2:43:43 PM:   "ruleId": "unexpected-character",
2:43:43 PM:   "source": "micromark-extension-mdx-jsx",
2:43:43 PM:   "url": "https://github.com/micromark/micromark-extension-mdx-jsx#unexpected-character-at-expected-expect"
2:43:43 PM: }

Reply:
okay, I will take a look.